sunpy/database/tests/test_tables.py

```diff
@@ -96,6 +113,108 @@ def test_tag_hashability():
 
 @pytest.mark.flaky(reruns=5)
 @pytest.mark.online
+def test_entries_from_fido_search_result(fido_search_result):
+    entries = list(entries_from_fido_search_result(fido_search_result))
+    # 65 entries for 8 instruments in fido_search_result
+    assert len(entries) == 65
+    # First 2 entries are from lyra
+    assert entries[0] == DatabaseEntry(
+        source='Proba2', provider='esa', physobs='irradiance',
+        fileid='http://proba2.oma.be/lyra/data/bsd/2012/01/01/lyra_20120101-000000_lev2_std.fits',
+        observation_time_start=datetime(2012, 1, 1, 0, 0),
+        observation_time_end=datetime(2012, 1, 2, 0, 0),
+        wavemin=np.nan, wavemax=np.nan,
+        instrument='lyra')
+    # 54 entries from EVE
+    assert entries[2] == DatabaseEntry(
+        source='SDO', provider='LASP', physobs='irradiance',
+        fileid='EVE_L1_esp_2012001_00',
+        observation_time_start=datetime(2012, 1, 1, 0, 0),
+        observation_time_end=datetime(2012, 1, 2, 0, 0),
+        instrument='EVE', size=-1.0,
+        wavemin=0.1, wavemax=30.400000000000002)
```

Comment:
really??? 30.400000000000000002 ??? Should we keep the rounding that was in the wavelength comparison?